climpred/checks.py

```diff
@@ -94,16 +94,19 @@ def is_initialized(obj, kind, what):
     return True
 
 
-def match_initialized_dims(init, ref):
+def match_initialized_dims(init, ref, uninitialized=False):
     """Checks that the reference matches all initialized dimensions except
-    for 'lead' and 'member'"""
+    for 'lead' and 'member'
```

Comment:
Text following """ usually are one line summaries; doesn't really matter tho: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

```
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """
```